GeneratedRegexPatterns/PreciselyClassifiedSecurityKeys.json

```diff
@@ -0,0 +1,404 @@
+    "Name": "AzureApimIdentifiableGatewayKey",
+    "DetectionMetadata": "Identifiable"
+  },
+  {
+    "Signatures": [
+      "APIM"
+    ],
+    "ChecksumSeeds": [
+      5145771916421312560
+    ],
+    "KeyLength": 64,
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=])(?<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{76}APIM[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{5}[AQgw]==)([^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=]|$)",
+    "RegexNormalizedSignature": "APIM",
+    "EncodeForUrl": false,
+    "Id": "SEC101/184",
+    "Name": "AzureApimIdentifiableRepositoryKey",
+    "DetectionMetadata": "Identifiable"
+  },
+  {
+    "Signatures": [
+      "AzCa"
+    ],
+    "ChecksumSeeds": [
+      4711718922539446320
+    ],
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=])(?<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{33}AzCa[A-P][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{5}=)([^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=]|$)",
+    "RegexNormalizedSignature": "AzCa",
+    "KeyLength": 32,
+    "EncodeForUrl": false,
+    "Id": "SEC101/154",
+    "Name": "AzureCacheForRedisIdentifiableKey",
+    "DetectionMetadata": "Identifiable"
+  },
+  {
+    "Signatures": [
+      "+ACR"
+    ],
+    "ChecksumSeeds": [
+      4702692889634567216
+    ],
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=])(?<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{42}\\+ACR[A-D][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/]{5})([^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890+/-_=]|$)",
+    "KeyLength": 39,
+    "RegexNormalizedSignature": "\\+ACR",
+    "EncodeForUrl": false,
+    "Id": "SEC101/176",
+    "Name": "AzureContainerRegistryIdentifiableKey",
+    "DetectionMetadata": "Identifiable"
+  },
+  {
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890])(?P<secret>secret_scanning_ab85fc6f8d7638cf1c11da812da308d43_[0-9A-Za-z]{5})([^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]|$)",
+    "Id": "SEC101/565",
+    "Name": "SecretScanningSampleToken",
+    "Signatures": [
+      "ab85"
+    ],
+    "DetectionMetadata": "HighEntropy, FixedSignature"
+  },
+  {
+    "Pattern": "(^|[^0-9a-z])(?<refine>oy2[a-p][0-9a-z]{15}[aq][0-9a-z]{11}[eu][bdfhjlnprtvxz357][a-p][0-9a-z]{11}[aeimquy4])([^aeimquy4]|$)",
+    "Id": "SEC101/030",
+    "Name": "NuGetApiKey",
+    "Signatures": [
+      "oy2"
+    ],
+    "DetectionMetadata": "HighEntropy, FixedSignature"
+  },
+  {
+    "Pattern": "^(?i)[a-z0-9.=\\-:[_@\\/*\\]+?]{32}$",
+    "Id": "SEC101/101",
+    "Name": "AadClientAppLegacyCredentials",
+    "Signatures": null,
+    "DetectionMetadata": "ObsoleteFormat, HighEntropy"
+  },
```

Comment:
Per my test, a timestamp like `2024-03-07T02:50:56.464790+00:00` would match this pattern as it's a simple string containing only numbers, letter `T` and symbols `-`, `:`, `.` and `+` while the length exactly equals 32